docker/mongodb/scripts/setup_user.sh

```diff
@@ -6,8 +6,13 @@ echo "************************************************************"
 echo "Setting up users..."
 echo "************************************************************"
 
+MONGO="mongo"
+if (( $(echo "$MONGO_MAJOR > 5.9" | bc -l) )); then
+    MONGO="mongosh"
+fi
```

Comment:
Can you pass it to the script instead of duplicating? Something like 
```
MONGO=$MONGO /home/mongodb/scripts/setup_user.sh
```

or just export $MONGO=...

Reply:
export!!